pkgs/servers/sql/postgresql/tsearch_extras/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, pkgs, fetchFromGitHub, pkgconfig, postgresql }:
+
+stdenv.mkDerivation rec {
+  name = "tsearch-extras-${version}";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "zulip";
+    repo = "tsearch_extras";
+    rev = "560e6748bd7cfcea98efd412ca80f08cff890f9c";
+    sha256 = "1ivg9zn7f1ks31ixxwywifwhzxn6py8s5ky1djyxnb0s60zckfjg";
+  };
+
+  buildInputs = [ postgresql pkgconfig ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -D tsearch_extras.so -t $out/lib/
+    install -D ./{tsearch_extras--1.0.sql,tsearch_extras.control} -t $out/share/extension
+  '';
+
+  meta = with stdenv.lib; {
+    description = "The package provides a few PostgreSQL functions that allow you to get at lower-level data about full text search";
```

Comment:
Also consider that the description must be short and to the point so that they render nicely in console output and the like; details go in `longDescription`.